crates/svm-sdk-tests/tests/endpoint/funding.rs

```diff
@@ -0,0 +1,70 @@
+use svm_sdk::storage::MockStorage;
+use svm_sdk::{app, Amount};
+
+use svm_sdk_tests::call_and_fund_1;
+
+#[app]
+mod App {
+    #[storage]
+    struct Storage {
+        coins: Amount,
+
+        called: bool,
+    }
+
+    #[fundable(update_coins)]
+    #[endpoint]
+    fn do_nothing() {}
```

Comment:
This `endpoint` returns `()` 
@moshababo

Reply:
Got it, 10x